makefile

```diff
@@ -212,7 +212,7 @@ Ccc  = $(strip $(CC) $(CFLAGS) $(GL_OPTS))
 ALL_SOURCE_FILES = \
  lib/gl/renderer.cpp lib/gl/renderer_core.cpp lib/gl/renderer_ff.cpp \
  lib/gl/types.cpp lib/aux_js.cpp lib/aux_vis.cpp lib/font.cpp lib/gl2ps.c \
- lib/material.cpp lib/openglvis.cpp lib/palettes.cpp lib/sdl.cpp \
+ lib/material.cpp lib/openglvis.cpp lib/palettes.cpp lib/sdl.cpp lib/stream_reader.cpp \
```

Comment:
These should be added to the cmake build as well.

Reply:
Thank you! Fixed.